packages/remark-lint-no-undefined-references/index.js

```diff
@@ -148,6 +152,26 @@
  *   1:8-1:18: Unexpected reference to undefined definition, expected corresponding definition (`mercury`) for a footnote or escaped opening bracket (`\[`) for regular text
  *
  * @example
+ *   {"gfm": true, "label": "input", "name": "gfm-table.md"}
+ *
+ *   | [Planet]            | [Radius]   |
+ *   |---------------------|------------|
+ *   | [Mercury]           | 2,439.7 km |
+ *   | [Venus][venus-docs] | 6,051.8 km |
+ *   | [Earth]             | 6,378 km   |
+ *
+ *   [Planet]: https://example.com/planet/
+ *   [Mercury]: https://example.com/mercury/
+ *   [Venus]: https://example.com/venus/
```

Comment:
All other tests are as minimal as can be. Is it needed to be this big, this many references? Or can it be smaller. Such as:

```suggestion
 *   | [Planet]          | [Radius]  |
 *   | ----------------- | --------- |
 *   | Mercury           | 2439.7 km |
 *
 *   [planet]: https://example.com/planet/
```

(Also some small changes such as padding and no casing in definition ID, which are the more common used styles in these projects!)

Reply:
I've simplified to almost this, with an extra link to Mercury to cover table cells (as well as headers)